### PR TITLE
14 componentes pacotes implantacao

### LINKED ISSUE #12: Diagrama de Implantação
## Descrição da Issue
Criação do diagrama  de Implantação
## Tasks:
- [ ] Esta bem documentado 
- [ ] As imagens condizem com o que e proposto para o projeto

## Critérios de Aceitação:
- [ ] Precisa de 5 revisores

## Assignees
A issue deve ser atribuída a pelo menos um colaborador do projeto.

## Labels
A issue deve ser marcada com uma ou mais tags adequadas, para fins de rastreamento do projeto.

## Milestone
A issue deve ser atribuída ao Milestone (sprint) correspondente previsto para sua execução.

### LINKED ISSUE #14: Diagrama de Componentes
## Descrição da Issue
Criação do diagrama  de Componentes
## Tasks:
- [ ] Esta bem documentado 
- [ ] As imagens condizem com o que e proposto para o projeto

## Critérios de Aceitação:
- [ ] Precisa de 5 revisores

## Assignees
A issue deve ser atribuída a pelo menos um colaborador do projeto.

## Labels
A issue deve ser marcada com uma ou mais tags adequadas, para fins de rastreamento do projeto.

## Milestone
A issue deve ser atribuída ao Milestone (sprint) correspondente previsto para sua execução.

### LINKED ISSUE #15: Diagrama  de Pacotes
## Descrição da Issue
Criação do diagrama  de Pacotes
## Tasks:
- [ ] Esta bem documentado 
- [ ] As imagens condizem com o que e proposto para o projeto

## Critérios de Aceitação:
- [ ] Precisa de 5 revisores

## Assignees
A issue deve ser atribuída a pelo menos um colaborador do projeto.

## Labels
A issue deve ser marcada com uma ou mais tags adequadas, para fins de rastreamento do projeto.

## Milestone
A issue deve ser atribuída ao Milestone (sprint) correspondente previsto para sua execução.

### LINKED ISSUE #16: 2° entrega de arquitetura, incluindo diagramas estáticos e dinâmicos.
## Descrição da Issue
Completar os documentos de diagramas tanto estáticos como dinâmicos para 2° entrega de Arquitetura e desenho de Software

## Diagrama estáticos 
- [x] Diagrama de Classes
- [x] Diagrama de Componentes
- [x] Diagrama de Implantação
- [x] Diagrama de Pacotes

## Diagrama dinâmicos
- [x] Diagrama de Atividades
- [x] Diagrama de Colaboração/Comunicação
- [x] Diagrama de Estados
- [x] Diagrama de Sequencia

## Extra
- [x] #23
- [ ] Adicionar atas das reuniões da Entrega 2
- [x] #24

## Critérios de Aceitação:
- [ ] finalizar todos os documentos corrigidos e se preparar para apresentação de arquitetura

## Assignees
A issue deve ser atribuída a pelo menos um colaborador do projeto.

## Labels
A issue deve ser marcada com uma ou mais tags adequadas, para fins de rastreamento do projeto.

## Milestone
A issue deve ser atribuída ao Milestone (sprint) correspondente previsto para sua execução.

### PR DESCRIPTION
**Pull request by Ericklevy**
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to <repo_name>?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the CONTRIBUTING.md doc.
- [ ] I have added necessary documentation (if appropriate).
- [ ] Any dependent changes have been merged and published in downstream modules.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by viniman27 — DISMISSED** [review]

**Review by viniman27 — COMMENTED** [review]
Tudo dentro dos conformes

**Review by JoaoPedro0803 — COMMENTED** [review]
Tudo ok!

**Review by GabrielRoger07 — COMMENTED** [review]
Revisado!

File: docs/Base/BPMN.md
Comment (klmurussi) [inline docs/Base/BPMN.md:1]: BPMN (Business Process Managemente Notation)
Essa sigla está correta? O M não seria de modelagem?

**Review by viniman27 — APPROVED** [review]

**Review by lxgustxl — APPROVED** [review]
Aprovado, sem erros aparentes.

**Review by GianMedeiros — APPROVED** [review]
Revisão aprovada sem defeitos ou modificações necessarias.

File: docs/Base/Rich-picture.md
Comment (klmurussi) [inline docs/Base/Rich-picture.md:1]: A professora tinha falado na entrega anterior que rich picture era mais um desenho e menos um fluxograma
Este rich picture me parece um fluxograma

**Review by klmurussi — APPROVED** [review]
Está tudo certo com diagrama de implantação, porém há problema com os artefatos que não fazem parte do escopo da branch (como o rich picture). O que pode dá problema ao dar merge.

**Comment by nickby2** [comment]
Está ótimo

**Review by nickby2 — APPROVED** [review]

**Comment by Ericklevy** [comment]
Mergado

**Pull request merged**